### PR TITLE
Added ability to add comment to a new Connection Request

### PR DESCRIPTION
**Pull request by bwitting**
via the Create Connection Request workflow action.

### CODE REVIEW

File: Rock/Workflow/Action/CreateConnectionRequest.cs [hunk 1]
```diff
@@ -129,6 +131,9 @@ public override bool Execute( RockContext rockContext, WorkflowAction action, Ob
                 }
             }
 
+            // Get the Comment
+            String comment = action.GetWorklowAttributeValue(GetAttributeValue(action, "ConnectionCommentAttribute").AsGuid());
```
Comment (nairdo) [inline Rock/Workflow/Action/CreateConnectionRequest.cs]: @bwitting Is the plan to use this comment as the connectionRequest.Comments property?  If so I think you forgot that part, right?

Comment (bwitting) [inline Rock/Workflow/Action/CreateConnectionRequest.cs]: D'oh. Yes.  Added now, thanks!

**Comment by JayGreentree** [comment]
@bwitting 
action.GetWorklowAttributeValue is missing the f .in workflow

**Comment by nairdo** [comment]
Yeah, but that's not Bronson's fault. We actually, accidentally implemented that method misspelled a while back ([in May 2014](https://github.com/SparkDevNetwork/Rock/commit/39b9969e3200a84c1c02f2d96e905c5e7817264c)).

**Pull request merged**